test-tools/wamr-ide/WASM-Source-Debug-Server/Docker/README.md

```diff
@@ -0,0 +1,16 @@
+### Build Docker Image
+
+-   execute `build_docker_image.bat` on `Windows`, note that change the `./resource/run.sh` and `./resource/debug.sh` to `LF` before building
```

Comment:
Can we provide run.bat and debug.bat for Windows, and let build_docker_image.bat invoke them? It is inconvenient for use to change the file content

Reply:
`run.sh` and `debug.sh` will be executed inside of `WASM source debug server` image rather than in host, so we don't need to provide the batch script for windows. Besides, change the `End of Line Sequence` to `LF` is just necessary when user want to build image on `windows` platform